compiler/src/iree/compiler/Codegen/Common/GPU/GPUNestedLayoutDistributionPatterns.cpp

```diff
@@ -1038,21 +1110,33 @@ struct DistributeContract final : OpDistributionPattern<vector::ContractionOp> {
     Type accElemTy = getElementTypeOrSelf(acc.getType());
 
     MLIRContext *ctx = contractOp.getContext();
-    Location loc = contractOp.getLoc();
 
     // Step 1: local contraction
     Value localInit = getCombiningIdentityValue(
         loc, rewriter, contractOp.getKind(), disAcc.getType());
-    vector::ContractionOp localContractOp = doDistributedContraction(
+    Value localContract = doDistributedContraction(
         rewriter, loc, ctx, contractOp, disLhs, disRhs, localInit);
 
+    // TODO: As per current upstream lowering implementations, there is no point
+    // in doing this because it does a select much later in a finer granularity
+    // rather than supporting predication. Moreover, since we are doing a select
+    // to cater reductions accross the distribution, we can choose not to mask
+    // the op post-distribution.
+
+    // if (mask) {
+    //   localContract =
+    //       vector::maskOperation(rewriter, localContract.getDefiningOp(),
+    //       mask)
+    //           ->getResult(0);
+    // }
+
```

Comment:
Instead of commenting out the code, can you write what we are choosing not to do instead?

Reply:
What we are choosing not to do is written in the comment above ?
(or Im not getting your ask)

Reply:
Sorry for being unclear, I was just saying that commented out code gets stale, so its better to write out what that code is supposed to do. Example change:

    // TODO: As per current upstream lowering implementations, there is no point
    // in masking the thread local contract because it does a select much later in a finer granularity
    // rather than supporting predication. Moreover, since we are doing a select
    // to cater reductions accross the distribution, we can choose not to mask
    // the op post-distribution.

Reply:
okay. now changed.